spec/request/api/v1/topic/topic_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Topic /api/v1/topics', type: :request do
+  let(:user) { create(:user, confirmed_at: Time.zone.today) }
+  subject do
+    10.times { create(:topic) }
+    login
+  end
+
+  it 'should return a successful response' do
+    subject
+    get '/api/v1/topics', headers: auth_headers
```

Comment:
and then define the call in a subject like out side the context
subject do
  get '/api/v1/topics', headers: auth_headers
end

and then call subject insted of get..